cli/enginecli/core.py

```diff
@@ -516,13 +498,15 @@ def lootbox_gogogo(
     )
 
     print("Setting pool controller...")
-    terminus_contract.set_pool_controller(
-        admin_token_pool_id, lootbox_contract.address, tx_config
+    admin_terminus_contract.set_pool_controller(
+        admin_terminus_pool_id, lootbox_contract.address, tx_config
```

Comment:
This seems a bit suspect to me - does the Lootbox contract give a way of minting these badges to someone?

Reply:
There is a method on lootbox called grant-admin-role that requires this. See ControllableWithTerminus.sol.